python/ray/train/torch/config.py

```diff
@@ -50,6 +51,9 @@ class TorchConfig(BackendConfig):
     def backend_cls(self):
         return _TorchBackend
 
+    def _repr_html_(self) -> str:
+        return make_table_html_repr(obj=self, title=type(self).__name__)
+
```

Comment:
Could we add this `BackendConfig` so it works with other frameworks?

Reply:
Yep, done.